roles/micado_master/start/templates/micado/micado-manifest.yml.j2

```diff
@@ -301,6 +301,8 @@ spec:
         volumeMounts:
         - name: submitter-out
           mountPath: /var/lib/micado/terraform/submitter
+        - name: wrapper-file
```

Comment:
What if there are other binaries in `/usr/local/bin` on the container - will they be overwritten?

Reply:
I am not sure. As the terraform binary is in /bin. If some other binary with the same name is there that will be the wrapper function right. Or are you suggesting to modify the path variable to make the wrapper script location first?